src/sqlfluff/core/parser/grammar/conditional.py

```diff
@@ -41,15 +41,16 @@ class Conditional(BaseGrammar):
     | "indentation" keys are currently set up.
     """
 
-    def __init__(self, *args, config_type: str = "indentation", **rules):
-        if not all(issubclass(arg, Indent) for arg in args):  # pragma: no cover
-            raise ValueError(
-                "Conditional is only designed to work with Indent segments."
-            )
-        if len(args) != 1:  # pragma: no cover
-            raise ValueError(
-                "Conditional is only designed to work with a single element."
-            )
+    def __init__(
+        self,
+        meta: Type[Indent],
+        config_type: str = "indentation",
+        **rules: Union[str, bool]
+    ):
+        assert issubclass(
+            meta, Indent
+        ), "Conditional is only designed to work with Indent/Dedent segments."
+        self._Meta = meta
```

Comment:
accidental capitalisation?

Reply:
I think I was trying to be clever to indicate that it's a class rather than an instance - but I think it might be more unhelpful. I'll revert to lowercase.